<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_12">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox_10">
             <property name="title">
              <string>相机显示</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_13">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QGroupBox{border:none}</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>54</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>帧率显示</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_9">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>120</height>
              </size>
             </property>
             <property name="title">
              <string>焊接控制</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_11">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>相机选择</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_14">
             <property name="title">
              <string>导轨控制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>触发控制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_8">
             <property name="title">
              <string>预处理</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox{border:none}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="5">
             <widget class="QPushButton" name="BN_Select_LaserLine">
              <property name="text">
               <string>选择激光线条（可选）</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="BN_Select_Board_noLaser">
              <property name="text">
               <string>选择无激光标定板</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="BN_Select_Board_Laser">
              <property name="text">
               <string>选择有激光标定板</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="BN_Select_Clib_Picture">
              <property name="text">
               <string>选择相机标定图片</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="BN_Select_Step_Pic">
              <property name="text">
               <string>选择步长计算图片</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="BN_Select_Base_Picture">
              <property name="text">
               <string>选择点云拼接基准图片</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox{border:none}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QListWidget" name="listWidget_CalibPic">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_BasePic"/>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_Board_noLaser"/>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_Board_Laser"/>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_Step_Pic"/>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_LaserLine"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox{border:none}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="2">
            <widget class="QRadioButton" name="radioButton_Y_add">
             <property name="text">
              <string>Y方向拼接</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="radioButton_X_add">
             <property name="text">
              <string>X轴方向拼接</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox_Methods">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="pushButton_Caculate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>仅计算标定参数</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>中心线提取算法：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QSpinBox" name="spinBox_6"/>
           </item>
           <item row="0" column="7" rowspan="2">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>55</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="9">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>卷积核Y标准差</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="spinBox_3"/>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox"/>
              </item>
              <item row="1" column="9">
               <widget class="QSpinBox" name="spinBox_5"/>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>卷积核X标准差</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinBox_2"/>
              </item>
              <item row="1" column="10">
               <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>计算阈值</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>卷积核X</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>卷积核Y</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QSpinBox" name="spinBox_4"/>
              </item>
              <item row="0" column="10">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>确定精度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="11">
               <widget class="QCheckBox" name="checkBox_3">
                <property name="text">
                 <string>亚像素</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>步长间隔：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>计算重建</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QVTKWidget" name="qvtkWidget"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>67</height>
           </size>
          </property>
          <property name="title">
           <string>CloudSetting</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="3" rowspan="2">
            <widget class="QSlider" name="horizontalSlider_Green">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="5" rowspan="2">
            <widget class="QSlider" name="horizontalSlider_Blue">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QSlider" name="horizontalSlider_Red">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="spinBox_green"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="Lable_Red">
             <property name="text">
              <string>Red:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="Lable_Size">
             <property name="text">
              <string>Size:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="spinBox_red"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="Lable_Green">
             <property name="text">
              <string>Green:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8" rowspan="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>恢复初试视角</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7" rowspan="2">
            <widget class="QSlider" name="horizontalSlider_Size">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QSpinBox" name="spinBox_size"/>
           </item>
           <item row="1" column="4">
            <widget class="QSpinBox" name="spinBox_blue"/>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="Lable_Blue">
             <property name="text">
              <string>Blue:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>361</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>相机标定误差</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_clearClib">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>清除标定信息</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>75</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>保存标定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>75</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>加载标定</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_isCamera">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>相机连接：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_isCalibed">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>标定：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_flagCalibed">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>No</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_flagCamera">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
